MekHQ/src/mekhq/campaign/mission/BotForce.java

```diff
@@ -115,7 +116,7 @@ public void setName(String name) {
     }
 
     public void setEntityList(ArrayList<Entity> entityList) {
-        this.entityList = entityList;
+        this.entityList = new ArrayList<>(entityList.stream().filter(x -> x != null).collect(Collectors.toList()));
```

Comment:
Could you add a comment here explaining what's being done? It looks like you're removing all null entities from the passed-in list, but it took me a minute to figure it out.

Reply:
Sure!